index.html

```diff
@@ -1017,6 +1017,206 @@ <h3>Interface Definition</h3>
+          <ol>
+            <li>Let <var>transport</var> be the <code><a>QuicTransportBase</a></code>
+            on which <code>getStats</code> is invoked.</li>
+            <li>If <var>transport</var>'s <a>[[\WebTransportState]]</a> is <code>"new"</code>
+            then abort these steps.</li>
+            <li>Return <var>p</var> and continue the following steps in the background.</li>
+            <ol>
+              <li>Gather the stats from the underlying QUIC connection.</li>
+              <li>Once stats have been gathered, resolve <var>p</var> with the
+              <code><a>QuicTransportStats</a></code> object, representing the gathered
+              stats</li>
+            </ol>
+          </ol>
+          <div>
+            <em>No parameters.</em>
+          </div>
+          <div>
+            <em>Return type:</em> <code><a>QuicTransportStats</a></code>
+          </div>
+        </dd>
+    </section>
+    <section id="quictransportstats*">
+      <h3><dfn>QuicTransportStats</dfn> Dictionary</h3>
+      <p>The <code>QuicTransportStats</code> dictionary includes information
+      QUIC connection level stats.
+      <div>
+        <pre class="idl">
+dictionary QuicTransportStats {
+  DOMHighResTimeStamp timestamp;
+  unsigned long long bytesSent;
+  unsigned long long packetsSent;
+  unsigned long long streamBytesSent;
+  unsigned long long streamBytesReceived;
+  unsigned long numOutgoingStreamsCreated;
+  unsigned long numIncomingStreamsCreated;
+  unsigned long long bytesReceived;
+  unsigned long long packetsReceived;
+  unsigned long long packetsProcessed;
+  unsigned long long bytesRetransmitted;
+  unsigned long long packetsRetransmitted;
+  unsigned long long packetsLost;
+  unsigned long long packetsDropped;
+  unsigned long long cryptoRetransmitCount;
+  unsigned long long minRttUs;
+  unsigned long long smoothedRttUs;
+  unsigned short maxPacketSize;
+  unsigned short maxReceivedPacketSize;
+  unsigned long estimatedBandwidthBps;
+  unsigned long long packetsReordered;
+  unsigned long long blockedFramesReceived;
+  unsigned long long blockedFramesSent;
+  unsigned long long connectivityProbingPacketsReceived;
+};</pre>
+        <section>
+          <h2>Dictionary <a class="idlType">QuicTransportStats</a> Members</h2>
+          <dl data-link-for="QuicTransportStats" data-dfn-for="QuicTransportStats" class=
+          "dictionary-members">
+            <dt><dfn><code>timestamp</code></dfn> of type <span class=
+            "idlMemberType"><a>DOMHighResTimeStamp</a></span></dt>
+            <dd>
+              <p>The <code>timestamp</code> for when the stats are gathered, relative to the
+              UNIX epoch (Jan 1, 1970, UTC)</p>
+            </dd>
+            <dt><dfn><code>bytesSent</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long long</a></span></dt>
+            <dd>
+              <p>The amount of bytes sent on the QUIC connection, including
+              retransmissions.</p>
```

Comment:
We should be more precise about how we count bytes.  Does this include UDP/IP/Ethernet framing?

Reply:
You're right we should specify this. I don't think this includes overhead. Comes from the WriteResult in quiche, tested here:
https://cs.chromium.org/chromium/src/net/quic/platform/impl/quic_socket_utils_test.cc?g=0&l=137

I updated the comment. Correct me if I'm wrong.